lib/jolie-js/src/jolie/js/JsUtils.java

```diff
@@ -224,4 +272,32 @@ public static void parseJsonIntoValue( Reader reader, Value value, boolean stric
 			throw new IOException( e );
 		}
 	}
+	
+	public static void parseNdJsonIntoValue( Reader reader, Value value, boolean strictEncoding )
+		throws IOException
+	{
+		String stringBuffer = new BufferedReader( (InputStreamReader) reader ).lines().collect( Collectors.joining( "\n" ) );
+		String[] stringItemVector = stringBuffer.split( "\n" );
```

Comment:
So first you collect all the lines in a stringBuffer, and then you split it again? Why don't you just use the stream given by `lines()` in buffered reader and its `forEach` method?